tests/test_measurements.py

```diff
@@ -73,6 +76,7 @@ def test_create_invalid_name(self):
         with self.assertRaises(KeyError):
             Measurement(metric, value, metric_set=self.metric_set)
 
+    @unittest.skip('FIXME DM8477 this is failing')
```

Comment:
Pedantic: "DM-8477" not "DM8477" (makes it easier to find things if we are consistent).